src/main/java/AnimalMain.java

```diff
@@ -1,10 +1,54 @@
 public class AnimalMain {
     public static void main(String[] args) {
+        /*
         Cat cat = new Cat();
         Dog dog = new Dog();
         cat.Swim(10);
         cat.Run(10);
         dog.Swim(150);
         dog.Run(200);
+        */
+
+        // create array of interfaces
+        InterfaceFigure  figures [] = new InterfaceFigure[3];
+
+        //added object to array by index
+        figures[0] = new Circle();
+        figures[1] = new Triangle();
+        figures[2] = new Square();
+
+        // start sum of all figures
+        double sAll = 0.0;
```

Comment:
sAll->sumAll, better use more characters for more clear name (understanding what is it)